data/remnant.txt

```diff
@@ -171,3 +241,13 @@ outfitter "Remnant"
 	"Cargo Scanner"
 	"Outfit Scanner"
 	"Tuning Rifle"
+
+outfitter "Remnant Salvage"
```

Comment:
I would be inclined to include korath in here to help differentiate if there is some other kind of salvage that becomes available later.

Reply:
I could, although I don't see much reason to split up salvage into separate categories.